src/components/PricePlan.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styled from 'styled-components';
+import { PlanType, PriceType } from '../types';
+import icon from '../assets/icon-music.svg';
+
+interface Props {
+  plan: PlanType;
+  selected: boolean;
+  onHandleClick: (val: string) => void;
```

Comment:
If the downstream variable is called selected, I would call the event propagating the variable onSelect. Its not that obvious but the reason is the as when you have a function called openFile, I would expect you to have another function called closeFile and not unsetFileHandle

Reply:
By changing this it now makes a lot more sense in the SummaryCard component as this triggers the setSelectedPlan.